### ISSUE TITLE
Windows & C++ Python

### ISSUE THREAD

**Issue by bmcgin** [theturtle32/WebSocket-Node#180]
This is a question, I am not sure i this is the appropriate place to ask...

I've been using the Windows  1.0.7 version without installing C++ or Python.  It's been working perfectly and has never once had a problem.  

With all of the work which has been done lately, I am considering installing the newest version and am wondering if C++ and Python are still optional.

**Comment by theturtle32** [comment]
I think it's still optional, but I don't have windows.  Please try it and let me know!  :)

**Comment by bmcgin** [comment]
I ran:  "npm update websocket"  and it installed v1.0.17.  Everything looks like it's running fine. I've tested sending data and receiving data via wss://.  All looks well and it's now running in production.  I'll let you know if anything goes haywire tomorrow as traffic picks up.

During the upgrade, it did not mention anything about native modules.   

Here's the npm output:

websocket@1.0.17 websocket
├── nan@1.0.0
├── debug@2.1.1 (ms@0.6.2)
└── typedarray-to-buffer@3.0.1 (is-typedarray@0.0.0)
